nixos/tests/k3s/default.nix

```diff
@@ -1,9 +1,21 @@
 { system ? builtins.currentSystem
 , pkgs ? import ../../.. { inherit system; }
+, lib ? pkgs.lib
 }:
+let
+  allK3s = {
```

Comment:
It is possible to infer this list from:
`builtins.readDir ../../../pkgs/applications/networking/cluster/k3s`
So it doesn't have to be changed over time.
I don't have a ready code to provide for this though.

Reply:
And `k3s` (with no version) doesn't have to be there because it could trigger an extra build? (I haven't checked, just wondering.) If version is already listed, it would be tested anyway.